commands/cluster-info.md

```diff
@@ -30,7 +29,50 @@ total_cluster_links_buffer_limit_exceeded:0
 * `cluster_stats_messages_received`: Number of messages received via the cluster node-to-node binary bus.
 * `total_cluster_links_buffer_limit_exceeded`: Accumulated count of cluster links freed due to exceeding the `cluster-link-sendbuf-limit` configuration.
 
-More information about the Current Epoch and Config Epoch variables are available in the Redis Cluster specification document.
+The following message-related fields may be included in the mapped reply if the value is not equal 0:
+
+```
+cluster_stats_messages_ping_sent:0
+cluster_stats_messages_pong_sent:0
+cluster_stats_messages_meet_sent:0
+cluster_stats_messages_fail_sent:0
+cluster_stats_messages_publish_sent:0
+cluster_stats_messages_auth-req_sent:0
+cluster_stats_messages_auth-ack_sent:0
+cluster_stats_messages_update_sent:0
+cluster_stats_messages_mfstart_sent:0
+cluster_stats_messages_module_sent:0
+cluster_stats_messages_publishshard_sent:0
+
+cluster_stats_messages_ping_received:0
+cluster_stats_messages_pong_received:0
+cluster_stats_messages_meet_received:0
+cluster_stats_messages_fail_received:0
+cluster_stats_messages_publish_received:0
+cluster_stats_messages_auth-req_received:0
+cluster_stats_messages_auth-ack_received:0
+cluster_stats_messages_update_received:0
+cluster_stats_messages_mfstart_received:0
+cluster_stats_messages_module_received:0
+cluster_stats_messages_publishshard_received:0
+```
+
+Each message type includes statistics on the number of messages sent and received.
+Here are the explanation of these message types:
+
+* `ping`: PING
```

Comment:
ping/pong/meet also contain gossip information.

Reply:
yes, should we mention that? or how should i mention that...